soccer/src/vision/ball/CameraBall.hpp

```diff
@@ -16,6 +19,13 @@ class CameraBall {
     CameraBall(RJ::Time timeCaptured, Geometry2d::Point pos)
         : timeCaptured(timeCaptured), pos(pos) {}
 
+    /**
+     * @brief Implicit conversion from DetectionBallMsg.
```

Comment:
This isn't an implicit conversion, there are multiple parameters...

That said, this is fine.

Reply:
Used to be, until I realized that it needed more information to be converted lmao